common/transport/src/main/proto/grpc/block_master.proto

```diff
@@ -69,6 +69,7 @@ message WorkerInfo {
   map<string, int64> capacityBytesOnTiers = 8;
   map<string, int64> usedBytesOnTiers = 9;
   optional BuildVersion buildVersion = 10;
+  optional WorkerIdentity identity = 11;
```

Comment:
If we treat this as the first version, we might consider to modify the first id directly.  
There is no backward compatible issue.
And is the first field still used for other part?

Reply:
@dbw9580  I am worrying this field is abandoned in the first day.